# HG changeset patch
# User Jorg K <jorgk@jorgk.com>
# Parent  62f95f3a771e9cd50fe88dc3d17ef919c2b7c6e2
Misc: Supply details-notification-incompatible-link string to avoid console logging its absence.

diff --git a/toolkit/locales/en-US/toolkit/about/aboutAddons.ftl b/toolkit/locales/en-US/toolkit/about/aboutAddons.ftl
--- a/toolkit/locales/en-US/toolkit/about/aboutAddons.ftl
+++ b/toolkit/locales/en-US/toolkit/about/aboutAddons.ftl
@@ -485,16 +485,17 @@ addon-page-options-button =
 
 ## Detail notifications
 ## Variables:
 ##   $name (String): name of the add-on.
 
 # Variables:
 #   $version (String): application version.
 details-notification-incompatible = { $name } is incompatible with { -brand-short-name } { $version }.
+details-notification-incompatible-link = More Information
 
 details-notification-unsigned-and-disabled = { $name } could not be verified for use in { -brand-short-name } and has been disabled.
 details-notification-unsigned-and-disabled-link = More Information
 
 details-notification-unsigned = { $name } could not be verified for use in { -brand-short-name }. Proceed with caution.
 details-notification-unsigned-link = More Information
 
 details-notification-blocked = { $name } has been disabled due to security or stability issues.
